modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/AnalyticsCloud/index.js

```diff
@@ -31,14 +41,26 @@ const ActivationStatusAnalyticsCloud = ({
 	userAccount,
 }) => {
 	const [{assetsPath}] = useCustomerPortal();
+	const [groupIdValue, setGroupIdValue] = useState();
 	const [activationStatusDate, setActivationStatusDate] = useState('');
 	const [isVisible, setIsVisible] = useState(false);
+	const [visible, setVisible] = useState(false);
 	const {observer, onClose} = useModal({
 		onClose: () => setIsVisible(false),
 	});
+	const {
+		observer: observerModalStatus,
+		onClose: onCloseModalStatus,
+	} = useModal({
```

Comment:
Declare a explicit name for for the first observer and onClose variables
```suggestion
	const {observer: ..., onClose: ...} = useModal({
		onClose: () => setIsVisible(false),
	});
	const {
		observer: observerModalStatus,
		onClose: onCloseModalStatus,
	} = useModal({
```